.github/workflows/schedule.cli.yml

```diff
@@ -10,30 +10,57 @@ permissions: read-all
 env:
   GH_TOKEN: ${{ secrets.CREATE_ISSUES }}
   ISSUE_REPOSITORY: ${{ github.repository }}
+  TITLE: 'schedule cli'
+  WORKFLOW: 'cli'
+  HEADER: 'cli'
 
 jobs:
   if-failed:
     runs-on: ubuntu-latest
     # See https://github.com/orgs/community/discussions/26238.
     steps:
-      - uses: actions/download-artifact@9782bd6a9848b53b110e712e20e42d89988822b7 # tag=v1.10.0
+      - name: Download artifact
+        uses: actions/github-script@v6
         with:
-          name: event_name
+          script: |
```

Comment:
This seems like a lot of complexity just so we can run these workflows in two files.

maybe you can consider abstracting out the code you want to avoid duplicating into another reusable workflow or action?

Reply:
the code to re-use is the one in pre-submit.cli.yml

Reply:
I could put the download script in an Action instead. Shall I do that now or wait to see if we have other use cases for it?

Reply:
We can wait for now.